kubernetes/data_source_kubernetes_persistent_volume.go

```diff
@@ -0,0 +1,96 @@
+package kubernetes
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func dataSourceKubernetesPersistentVolume() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceKubernetesPersistentVolumeRead,
+
+		Schema: map[string]*schema.Schema{
+			"metadata": metadataSchema("persistent volume", false),
+			"spec": {
```

Comment:
Are there any differences that would warrant the schema of the datasource to not be shared with the resource?
It would be preferable to share schema between resource and datasource whenever possible to minimize the potential for drift.